src/OrbitGl/App.cpp

```diff
@@ -151,6 +154,41 @@ PresetLoadState GetPresetLoadStateForProcess(
 
   return PresetLoadState::kPartiallyLoadable;
 }
+
+orbit_metrics_uploader::CaptureStartData CreateCaptureStartData(
+    const std::vector<FunctionInfo>& all_instrumented_functions, int64_t number_of_frame_tracks) {
+  orbit_metrics_uploader::CaptureStartData capture_start_data{};
+
+  for (const auto& function : all_instrumented_functions) {
+    switch (function.orbit_type()) {
+      case orbit_client_protos::FunctionInfo_OrbitType_kNone:
```

Comment:
All of them are instrumented functions?

Reply:
The proto field `number_of_instrumented_functions` has the following comment:
```
  // Number of functions the user dynamically instrumented (hooked) before the
  // start of this capture. This does not include functions that are
  // instrumented via the manual instrumentation api (Orbit.h).
```

But I see how this still might be confusing. Do you think maybe the name `number_of_dynamically_instrumented_functions` is better? Or any other suggestions. 

There is also going to be a different field soon:
```
  // Total number of timers of all dynamically instrumented (hooked) functions
  // that were recorded in this capture. This does not include anything from
  // the manual instrumentation api (Orbit.h).
  int64 number_of_instrumented_function_timers = 10;
```
I could also call this `number_of_dynamically_instrumented_function_timers`

Reply:
Yes, I will rename it in every place you like.

> I could also call this number_of_dynamically_instrumented_function_timers

And yes, to not confuse with the manual ones, thanks :)

Reply:
As discussed offline, sadly not possible anymore, since this is only a copy from the source of truth